util/send-payload-pls.sh

```diff
@@ -1,17 +1,91 @@
 #!/bin/bash
 #
-# Script to post a payload against a local webserver at each paranoia level
+# Script to post a payload against a local webserver at each paranoia level.
 #
-# Note: Webserver has to be prepared to take desired PL as Request Header "PL"
+# Note: Webserver has to be prepared to take desired PL as Request Header "PL".
+
+# WARNING: Setting the paranoia level using a header without proper authentication and authorization
+# is extremely dangerous, and is not recommended for production.
+
 # Check the access log format at https://www.netnea.com/cms/apache-tutorial-5_extending-access-log/
+# This script assumes %{ModSecAnomalyScoreIn}e is the last access log column
+# For better results use the collaborative mode and set the SecDefaultAction to 'pass'
+# 
+# The anomaly score envvar can be set as follows:
+# SecAction "id:90101,phase:5,pass,nolog,setenv:ModSecAnomalyScoreIn=%{TX.anomaly_score}"
```

Comment:
If you describe it like this, you should also describe how to display this in the access log.

Problem: You can not display this in the access log on nginx. There you need to write a custom rule that contains the scores in an alert message.

Reply:
Do you have a sample of the NGNIX rule?

Reply:
See this article: https://www.oreilly.com/ideas/how-to-tune-your-waf-installation-to-reduce-false-positives 
Rule 980145.